components/form/demo/advanced-search-form.md

```diff
@@ -114,7 +114,7 @@ ReactDOM.render(
   border-radius: 6px;
 }
 
-/* becasue the label width is not definite, you need to adjust it to center by yourself */
+/* because the label width is not fixed, you may need to adjust it */
```

Comment:
@GrayChoi Hi Gray, I played a little bit with the demo, so now I understand what you mean, however, adjusting the left edge of the form row using `left: -6` does not really center the form no matter the width of the viewport.